website/docs/reference/resource-properties/constraints.md

```diff
@@ -350,7 +350,62 @@ models:
 
 </File>
 
-Expected DDL to enforce constraints:
+### Column-level constraint on nested column:
```

Comment:
This functionality is only supported for BigQuery at the moment - could we make it clear that that's the case?

Reply:
@MichelleArk this example is in a tab dedicated to BigQuery only